examples/drug_gripnet/README.md

```diff
@@ -1,13 +1,39 @@
-# Link Prediction: GripNet Framework on PoSE Sample Dataset
+# Polypharmacy Side Effect Prediction using GripNet (Link Prediction)
 
 ## 1. Description
-This example is constructed by [GripNet Framework](https://arxiv.org/abs/2010.15914) on Polypharmacy Side Effect (PoSE) prediction task. The PoSE sample dataset is from BioSNAP-Decagon dataset and the target is to predict relations between drug nodes on the whole gene-drug heterogeneous graph.
+
+Polypharmacy is the concurrent use of multiple medications by patients. Its side effects, polypharmacy side effects, are caused by drug combinations rather than by any single drug. In this example, we train a GripNet model [1] for predicting polypharmacy side effects.
```

Comment:
e.g. add hyperlink to GripNet here or in Sec. 2 and move the references to the bottom.